app.js

```diff
@@ -22,48 +32,69 @@ var books = [{
         author: "Средний"
     }
 ];
-
+*/
 
 app.get('/', (req, res) => {
     res.send('Hello api');
 });
 
 app.get('/books', (req, res) => {
-    res.send(books);
+    //res.send(books);
+    Book.find((err, book) => {
+        if (err) throw err;
+        return res.send(book);
+    })
 });
 
-app.get('/books/:id', (req, res) => {
+/*app.get('/books/:id', (req, res) => {
     var book = books.find((book) => {
         return book.id === Number(req.params.id);
     });
     res.send(book);
-});
+});*/
 
 app.post('/books', (req, res) => {
-    var book = {
+    /*var book = {
         id: Date.now(),
```

Comment:
Timestamp is a nice solution for an `id` field, but MongoDB has it's own ids. If you want to generate ids by yourself, please use hash functions.